LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/LabConstants.java

```diff
@@ -94,7 +94,7 @@ static final class FederationProvider {
         public static final String CIAM = "ciam";
     }
 
-    static final class AzureEnvironment {
+    public static final class AzureEnvironment {
```

Comment:
Was this change actually needed?

Reply:
Yes. In the other PR, I used this to check (against AzureEnvironment of the account) if we should be making a drs discovery request to prod or ppe.